morpheus/_lib/src/utilities/cupy_util.cpp

```diff
@@ -80,8 +80,11 @@ pybind11::object CupyUtil::tensor_to_cupy(const TensorObject& tensor)
 
     auto ptr    = (uintptr_t)tensor.data();
     auto nbytes = tensor.bytes();
-    auto owner  = py_tensor;
-    int dev_id  = -1;
+
+    DCHECK(nbytes > 0);
+
+    auto owner = py_tensor;
+    int dev_id = -1;
```

Comment:
```suggestion
    auto owner = py_tensor;
    int dev_id = -1;
```

Isnt `nbytes` a `size_t` now? So it can never be negative?